numba/tests/cffi_usecases.py

```diff
@@ -51,15 +53,19 @@
     ffi_ool = FFI()
     ffi.set_source('cffi_usecases_ool', source)
     ffi.cdef(defs, override=True)
-    ffi.compile()
-
-    import cffi_usecases_ool
-    cffi_support.register_module(cffi_usecases_ool)
-    cffi_sin_ool = cffi_usecases_ool.lib.sin
-    cffi_cos_ool = cffi_usecases_ool.lib.cos
-    cffi_foo = cffi_usecases_ool.lib.foo
-    vsSin = cffi_usecases_ool.lib.vsSin
-    vdSin = cffi_usecases_ool.lib.vdSin
+    tmpdir = static_temp_directory('test_cffi')
+    ffi.compile(tmpdir=tmpdir)
```

Comment:
As a sidenote, launching a compiler on module import may not be the best idea, if e.g. the user is introspecting modules :-)